README.md

```diff
@@ -16,50 +30,72 @@ If you'd like to set up your own copy of the SIMPLE database, here's what we rec
 1. Clone or download a copy of this repo somewhere locally
 
 2. Set up an environment for the python code. 
-A conda environment file exists for convenience.
-
-3. Install the AstrodbKit2 package:
-
-```bash
-pip install git+https://github.com/dr-rodriguez/AstrodbKit2
-```
-
-5. Create the database. If you already have a database (eg, a binary db file with SQLite), **skip to the next step**. 
-
-```python
-from astrodbkit2.astrodb import create_database
-from simple.schema import *
-
-connection_string = 'sqlite:///SIMPLE.db'  # SQLite
-create_database(connection_string)
-```
-
-6. Connect to your existing database.
-
-```python
-from astrodbkit2.astrodb import Database
-
-connection_string = 'sqlite:///SIMPLE.db'  # SQLite
-db = Database(connection_string)
-```
-
-7. Load database contents from source, if needed. 
+A conda environment file exists for convenience. The following commands will create and activate an environment called `simple-db`:
+
+    ```bash
+    conda env create -f environment2.yml
+    conda activate simple-db
+    ```
+    <!--- 
```

Comment:
Why not delete all this commented code? It will still be in the version history should we need it.